lib/src/test/kotlin/network/account/AccountTest.kt

```diff
@@ -766,12 +756,10 @@ class AccountTest {
     fun `test udc deploy with parameters`() {
         assumeTrue(NetworkConfig.isTestEnabled(requiresGas = true))
 
-        assumeFalse(network == Network.GOERLI_INTEGRATION)
+        assumeFalse(network == Network.SEPOLIA_INTEGRATION)
```

Comment:
Is this still needed?

Previously, it was used because (for whatever reason) we could only run the test on `GOERLI_INTEGRATION`. Does that still hold and this test pass **only** on `SEPOLIA_INTEGRATION`? 

The same applies to all the other `assumeFalse` / `assumeTrue` checks